packages/docs/docs/questionnaires/structured-data-capture.md

```diff
@@ -0,0 +1,468 @@
+    },
+    {
+      "linkId": "phone",
+      "item": [
+        { "linkId": "number", "answer": [{ "valueString": "+1 800 123 4567" }] },
+        { "linkId": "use", "answer": [{ "valueCoding": { "code": "work" } }] }
+      ]
+    }
+  ]
+}
+```
+
+```json
+{
+  "resourceType": "Patient",
+  // No values for name provided in the response,
+  // so the field is omitted entirely
+  "telecom": [
+    {
+      "use": "home", // Default value from template used when no result for item
+      "system": "phone",
+      "value": "+1 555 555 5555"
+    },
+    {
+      "use": "work",
+      "system": "phone",
+      "value": "+1 800 123 4567"
+    }
+  ]
+}
+```
+
+[extract-template]: https://build.fhir.org/ig/HL7/sdc/extraction.html#template-extract
+[extract-ext]: https://build.fhir.org/ig/HL7/sdc/StructureDefinition-sdc-questionnaire-templateExtract.html
+[value-ext]: https://build.fhir.org/ig/HL7/sdc/StructureDefinition-sdc-questionnaire-templateExtractValue.html
+[context-ext]: https://build.fhir.org/ig/HL7/sdc/StructureDefinition-sdc-questionnaire-templateExtractContext.html
+[fhirpath]: https://hl7.org/fhir/fhirpath.html
+
+## Extraction API
+
+When responses to the annotated Questionnaire are received, they can be passed to the [`$extract` API][extract] to be
+parsed into resources.
+
+<MedplumCodeBlock language="ts" selectBlocks="get">
+  {ExampleCode}
+</MedplumCodeBlock>
+
+Alternatively, the questionnaire and response can be passed in directly, in case they are not stored on the Medplum
+server:
+
+<MedplumCodeBlock language="ts" selectBlocks="post">
+  {ExampleCode}
+</MedplumCodeBlock>
+
+[extract]: /docs/api/fhir/operations/extract
+
+## Results Bundle
+
+The resources populated with values parsed from the `QuestionnaireResponse` are returned from the `$extract` operation
+in a transaction [`Bundle`][bundle]:
+
+```json
+{
+  "resourceType": "Bundle",
+  "type": "transaction",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:4a51e8bd-0170-412d-a78c-0ca2fb174dca",
+      "request": { "method": "POST", "url": "Patient" },
+      "resource": {
+        "resourceType": "Patient",
+        "name": { "text": "John Doe" },
+        "telecom": [{ "use": "home", "system": "phone", "value": "+1 555 555 5555" }]
+      }
+    }
+  ]
+}
+```
+
+This can be uploaded via the [Batch API][batch] to save the generated resources to the Medplum server.
+
+[bundle]: /docs/api/fhir/resources/bundle
+
+## Parsing Multiple Resources
+
+Many questionnaires gather information that must be processed into multiple separate FHIR resources, often ones that
+link together via references. To [link the resources together][bundle-intern-refs] in the results `Bundle`, place one
+or more [`extractAllocateId`][allocate-id] extensions at the root of the `Questionnaire` to pregenerate an internal
+reference string that can be assigned to a template during extraction via the associated `templateExtract` extension
+and then referenced in other resource templates.
+
+```json
```

Comment:
I think I understand what's going on here, but it introduces 2 concepts at the same time:

1. Setting the `sdc-questionnaire-templateExtract` at a non-root item
2. Allocating a reference

This is a complicated topic, so having these split into 2 separate examples would help follow along, even if they are a bit contrived.

For example: 
1. Take patient name for multiple patients (e.g. siblings) - no dependencies
2. An example like this that creates 2 resources that are linked, liek this